jekyll/_cci2/machine-runner-3-manual-installation.adoc

```diff
@@ -65,7 +65,7 @@ NOTE: The current CircleCI machine runner binary can always be found by using `c
 [,shell]
 ----
 export RUNNERVERSION='current'
-export CPUARCH=$(/usr/bin/arch | grep 'x86_64' && echo 'amd64' || echo 'arm64')
+export CPUARCH=$(/usr/bin/arch | grep 'x86_64' >/dev/null 2>/dev/null && echo 'amd64' || echo 'arm64')
```

Comment:
```suggestion
export CPUARCH=$(/usr/bin/arch | grep 'x86_64' >/dev/null 2>&1 && echo 'amd64' || echo 'arm64')
```

Reply:
The use of `&1` isn't posix compliant for shell redirect and will cause issues with certain shells.